src/__tests__/options/getPackageOptions.test.ts

```diff
@@ -0,0 +1,53 @@
+import { cosmiconfigSync } from 'cosmiconfig';
+import { getPackageOptions } from '../../options/getPackageOptions';
+import { PackageOptions } from '../../types/BeachballOptions';
+
+jest.mock('cosmiconfig');
+
+describe('getPackageOptions', () => {
+  const defaultConfig: PackageOptions = {
+    gitTags: true,
+    defaultNpmTag: 'latest',
+    disallowedChangeTypes: ['major'],
+    tag: null,
+  };
+  const mockCosmiconfig = (options: Partial<PackageOptions> = {}, result: any = {}) => {
+    (cosmiconfigSync as jest.Mock).mockReturnValue({
+      load: (): any => ({
+        filepath: '',
+        config: {
+          ...defaultConfig,
+          ...options,
+        },
+        ...result,
+      }),
+    });
+  };
+
+  it('should return config if defined', () => {
+    // Arrange
```

Comment:
I'm quite a firm believer in them for readability purposes, but i see that they are not in other unit tests, can remove

Reply:
FWIW I'm fine with them in or removed. It won't hurt to have them in but at the same time they don't bring much value indeed. It's up to you @ling1726.

Reply:
**thought:** in reality there are no style-guides in this repo, so you can do whatever you like IMO.

On the other hand  triple AAA is a fundamental knowledge for any kind of testing in any language so this might feel like saying the obvious ( something like writing comments that a "Function that it is a function".

↓
```
// define function
function doWork(){}
```

Next thing is, that comments get outdated right after you wrote them down (most of the time from my experience), thus if you wanna express intent (write readable code), it should be reflected within naming. 

For testing there is a well known "recommendation" to write your tests in following manner:
> NOTE: those comments are not part of "the code" and aI added them only for demonstration purposes

```
it(`should....`, () =>{ 
// here is your arrange, explicit intent by reading through code
const {sum} = setup()

// here is your act, explicit intent by reading through code
const actual = sum(5,2) 
const expected = 10

// here is your assertion 
expect(actual).toBe(expected)
})
```

Reply:
I don't have super strong feelings on the comments, but my inclination would be to remove them since each test is short enough that it's easy to tell what's going on. And agreed with @Hotell about conveying intent through naming in this type of situation.